Library/Formula/gobject-introspection.rb

```diff
@@ -16,7 +16,9 @@ class GobjectIntrospection < Formula
 
   depends_on "pkg-config" => :run
   depends_on "glib"
+  depends_on "cairo"
   depends_on "libffi"
+  depends_on :python
```

Comment:
http://bot.brew.sh/job/Homebrew%20Pull%20Requests/33685/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_gobject_introspection/

Well, it's a bug somewhere.